server/routes/index.js

```diff
@@ -10,6 +11,8 @@ router.get("/welcome", function (req, res, next) {
 router.post('/register', auth.register);
 router.post('/login', auth.login);
 
+router.get('/users/:id', users.getUser)
+
```

Comment:
i think these routes will change once my ProjectAPI brange gets merged. itll be router.use instead and itll import the auth and user routes from a different file within this routes directory

Reply:
totally fine. I was thinking of it as sort of a temporary thing because I knew you were working on the route that returns the user with projects.